ShoppingCart/Discount.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace ShoppingCart
+{
+    public class Discount : IDiscount
+    {
+        public static double discountPercentage = 5;
+
+        public static Dictionary<string, double> CategorialDiscount = new Dictionary<string, double>();
```

Comment:
This should not be public property

Reply:
needs to be public as it has to be referenced elsewhere